src/index.js

```diff
@@ -1,2 +1,7 @@
-export {default as RandomForestClassifier} from './RandomForestClassifier';
-export {default as RandomForestRegression} from './RandomForestRegression';
+import {default as RandomForestClassifier} from './RandomForestClassifier';
+import {default as RandomForestRegression} from './RandomForestRegression';
+
+export {
```

Comment:
you should always avoid this, use this as inspiration of how to fix it
https://github.com/mljs/decision-tree-cart/commit/4ceff099a66717a08937086564b3691fe20aad05